### PR TITLE
fix(osaka): make it compatable with the new papi version

### PR DESCRIPTION
**Pull request by yehudagoldner**
affects: @haaretz/htz-components

**Table of Contents**

- [Description](#description)
- [Checklist](#checklist)

**Related issue(s):** #0

## Description

**Notes:**

## Checklist

Design:
  * [ ] Approved by designer

Usability:
  * [ ] Developer
  * [ ] Project manager

Tests
  
  * [ ] UI snapshot testing 
  * [ ] Unit testing
  * [ ] Integration testing
  * [ ] E2E testing

### CODE REVIEW

File: packages/components/htz-components/src/components/Osaka/OsakaController.js [hunk 1]
```diff
@@ -43,7 +43,8 @@ class OsakaWithOutbrain extends React.Component {
     // eslint-disable-next-line react/prop-types
     const { promotedElement, hostname, canonicalUrl, } = this.props;
     const url = canonicalUrl ? this.changeSubDomain(canonicalUrl) : '';
-    const promoted = promotedElement[0].banners[0];
+    console.log(5424532, this.props);
```
Comment (TxHawks) [inline packages/components/htz-components/src/components/Osaka/OsakaController.js]: This was left here by mistake, right?

File: packages/components/htz-components/src/components/Osaka/OsakaController.js [hunk 2]
```diff
@@ -43,7 +43,8 @@ class OsakaWithOutbrain extends React.Component {
...
+    const promoted = promotedElement[0].content.banners[0];
```
Comment (TxHawks) [inline packages/components/htz-components/src/components/Osaka/OsakaController.js]: Is this safe? Is there always an element `0` in the `promotedElement` array? Is it indeed always an array?

Comment (yehudagoldner) [reply packages/components/htz-components/src/components/Osaka/OsakaController.js]: the previus also reffered to element 0 and it worked

Comment (TxHawks) [reply packages/components/htz-components/src/components/Osaka/OsakaController.js]: I realize, but that doesn't mean it's always safe. Is it?

File: packages/components/htz-components/src/components/Osaka/OsakaController.js (hunk 1 above)
Comment (yehudagoldner) [reply packages/components/htz-components/src/components/Osaka/OsakaController.js]: yes, sorry...

File: packages/components/htz-components/src/components/Osaka/OsakaController.js (hunk 2 above)
Comment (yehudagoldner) [reply packages/components/htz-components/src/components/Osaka/OsakaController.js]: well, it worked so far :) 
i will verify the element exist anyway

**Pull request closed without merging**